app/src/main/java/changhwan/experiment/sopthomework/BindingAdapters.kt

```diff
@@ -0,0 +1,16 @@
+package changhwan.experiment.sopthomework
+
+
+import android.graphics.drawable.Drawable
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+
+
+object BindingAdapters {
+
+    @JvmStatic
+    @BindingAdapter("imgRes")
+    fun setImage (imageview : ImageView, resid : Int){
+        imageview.setImageResource(resid)
+    }
```

Comment:
바인딩 어댑터의 함수의 경우
```suggestion
    fun ImageView.setImage (resid : Int){
        setImageResource(resid)
    }
```
이런식으로 확장함수로 표현할 수 있습니다~!!